roles/confluent.schema-registry/templates/schema-registry_kerberos.properties.j2

```diff
@@ -0,0 +1,18 @@
+# Maintained by Ansible
+kafkastore.bootstrap.servers={% for inventory_hostname in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_PLAINTEXT://{{ inventory_hostname }}:{{broker.config.port}}{% endfor %}
+
+kafkastore.connection.url=zookeeper.connect={% for inventory_hostname in groups['zookeeper'] %}{% if loop.index > 1%},{% endif %}{{ inventory_hostname }}:{{zookeeper.config.clientPort}}{% endfor %}
+
+{% include './includes/base_schema-registry.properties.j2' %}
+
+kafkastore.security.protocol=SASL_PLAINTEXT
+kafkastore.sasl.mechanism=GSSAPI
+
+kafkastore.sasl.kerberos.service.name=kafka
+
+# Kerberos Properties
+kafkastore.sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required \
+   useKeyTab=true \
+   storeKey=true \
+   keyTab="{{schema.registry.keytab}}/schemaregistry-{{inventory_hostname}}.keytab" \
```

Comment:
document keytab file naming convention, how would you know its schemaregistry-{{inventory_hostname}}.keytab and not schema-registry-{{inventory_hostname}}.keytab

Reply:
created https://confluentinc.atlassian.net/browse/PLAT-602 to track

Reply:
I've filed a docs jira for this.